### ISSUE TITLE
get_frame_locals doesn't use the keyword max_var_size

### ISSUE THREAD

**Issue by grzn** [getsentry/raven-python#679]
From utils/stacks.py:

```
def get_frame_locals(frame, transformer=transform, max_var_size=4096):
    f_locals = getattr(frame, 'f_locals', None)
    if not f_locals:
        return None

    if not isinstance(f_locals, dict):
        # XXX: Genshi (and maybe others) have broken implementations of
        # f_locals that are not actually dictionaries
        try:
            f_locals = to_dict(f_locals)
        except Exception:
            return None

    f_vars = {}
    f_size = 0
    for k, v in six.iteritems(f_locals):
        v = transformer(v)
        v_size = len(repr(v))
        if v_size + f_size < 4096:
            f_vars[k] = v
            f_size += v_size
    return f_vars
```

In line 184 it uses the hard-coded value 4096 and not the keyword argument max_var_size.

**Comment by tilgovi** [comment]
This looks to have been fixed in 1a23de64bedd591f561eef1c71e03a30f045cc3b

**Comment by ashwoods** [comment]
@tilgovi sweet, thx for pointing it out.
